nailgun/entities.py

```diff
@@ -1670,12 +1670,21 @@ def create_missing(self):
                 'Found instance attributes: {0}'.format(self.get_values())
             )
         super(Host, self).create_missing()
+        self.name = self.name.lower()
```

Comment:
Can you link to [BZ 1227854](https://bugzilla.redhat.com/show_bug.cgi?id=1227854), please? Something like this is fine:

```python
        # See: https://bugzilla.redhat.com/show_bug.cgi?id=1227854
        self.name = self.name.lower()
```